.github/workflows/release.yaml

```diff
@@ -16,7 +16,7 @@ env:
   # /dev.Dockerfile
   # /make/builder.Dockerfile
   # /.github/workflows/main.yml
-  GO_VERSION: 1.15.6
+  GO_VERSION: 1.15.7
```

Comment:
There's also a go version in `main.yml` that should be updated.

Reply:
Updated to bump the version in `lnrpc/Dockerfile` as well.